packages/flutter/lib/src/material/tabs.dart

```diff
@@ -34,23 +34,35 @@ const double _kMaxTabWidth = 264.0;
 ///  * [TabController], which coordinates tab selection between a [TabBar] and a [TabBarView].
 ///  * <https://material.google.com/components/tabs.html>
 class Tab extends StatelessWidget {
-  /// Creates a material design [TabBar] tab. At least one of [text] and [icon]
-  /// must be non-null.
+  /// Creates a material design [TabBar] tab. At least one of [text], [icon],
+  /// and [child] must be non-null. [text] and [child] cannot be used at the
```

Comment:
Avoid starting sentences with lower case letters by phrasing this "The [text] and [child] arguments cannot..."

Reply:
Also avoid saying "cannot". They can, it will just throw. Prefer "must not".